src/EnergyPlus/Psychrometrics.hh

```diff
@@ -284,7 +282,7 @@ namespace Psychrometrics {
 
 		// REFERENCES:
 		// see ASHRAE Fundamentals Psychrometric Chapter
-		// USAGE:  hg = PsyHgAirFnWTdb(w,T)
+		// USAGE:  hg = PsyHgAirFnWTdb(T)
```

Comment:
@mjwitte As I see it, this function has been wrong the whole time then.  It should always have been named ```PsyHgAirFnTdb``` since it is deceiving to devs that it is actually a function of w.  I will process the source and change the function name to remove W.  This is a good thing to note.

If, later, we implement an actual psych function that is dependent on w, we can add it as another function with W in the title.